style.css

```diff
@@ -1,25 +1,27 @@
-/*
-color: #962D77;
-color: #DEB995;
-color: #FF9D00;
-color: #FF7B00;
-color: #612000;
- */
-
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+html {
+  scroll-behavior: smooth;
```

Comment:
wow, I didn't knew this one! Game changer for me 😮

Reply:
I know, right? But please, be careful, because browser support is still not so great with this one. I've decided to use it, because I want to avoid any JS in this project, or at least at this stage; and I'm fine with the fallback. If it works - it's great, but if not - the functionality is still there, just no glam.